RELEASES.md

```diff
@@ -5,6 +5,7 @@
 * - IMPORTANT: `eth_getProof` calls for historical state will be rejected by default.
   - On archive nodes (`"pruning-enabled": false`): queries for historical proofs for state older than approximately 24 hours preceding the last accepted block will be rejected by default. This can be adjusted with the new option `historical-proof-query-window` which defines the number of blocks before the last accepted block which should be accepted for state proof queries, or set to `0` to accept any block number state query (previous behavior).
   - On `pruning` nodes: queries for proofs past the tip buffer (32 blocks) will be rejected. This is in support of moving to a path based storage scheme, which does not support historical state proofs.
+* Added `Fortuna` upgrade as optional and defaulting to a `nil` timestamp.
```

Comment:
Should we mention this anyway, given

> Fortuna have no effect on subnet-evm networks

This does feel like maintenance/code-similarity-for-coreth and I don't think users would care about it really?

Reply:
I think we still want to mention this as they will see it startup messages

Reply:
I will put up a more descriptive note once we release a new version after avago mainnet release